tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -236,6 +264,10 @@ def open_browsing_privacy_settings(self):
         self._tap_menu_item(self._browsing_privacy_item_locator)
         return BrowsingPrivacy(self.marionette)
 
+    # method to open menu item that does not return object
+    def simple_open_menu(self, menu_locator):
```

Comment:
Following up on IRC. I think we can factorize all the `open_*_settings()` like this:

```python
     def open_battery_settings(self):
         return self._open_subpage_settings('Battery')

     def open_wifi_settings(self):
         return self._open_subpage_settings('Wifi')

     def _open_subpage_settings(self, class_name, locator):
         self._tap_menu_item(locator)
         Class = self._get_class_by_name(class_name):
         return Class(self.marionette)

    def _get_class_by_name(self, class_name):
         package_path = 'gaiatest.apps.settings.regions.{}'.format(class_name.lower())
         mod = __import__(package_path, fromlist=[class_name])
         Class = getattr(mod, class_name)
         return Class
```